elisp/guix-deriviation.el

```diff
@@ -0,0 +1,89 @@
+;;; guix-deriviation.el --- Guix deriviation mode
+
+;; Copyright © 2017 Oleg Pykhalov <go.wigust@gmail.com>
+;; Copyright © 2017 Alex Kost <alezost@gmail.com>
```

Comment:
This line shouldn't be here, since I have not done anything in this file yet :-)